src/libman/Commands/UninstallCommand.cs

```diff
@@ -116,11 +115,9 @@ public override BaseCommand Configure(CommandLineApplication parent)
                 throw new InvalidOperationException(string.Join(Environment.NewLine, errors));
             }
```

Comment:
Isn't all this validation supposed to be at Manifest? I believe we have it all covered already

Reply:
This are command line parameter validations.